snap/snapcraft.yaml

```diff
@@ -40,6 +40,12 @@ package-repositories:
     key-id: 6676E3F1A76ADBE4488944BF7D1A96D4BF78C79E
     url: |
         https://ppa.launchpadcontent.net/data-platform/percona-backup-mongodb/ubuntu
+  - type: apt
+    components: [main]
+    suites: [jammy]
+    key-id: 6676E3F1A76ADBE4488944BF7D1A96D4BF78C79E
+    url: |
+        https://ppa.launchpadcontent.net/data-platform/mongodb-exporter/ubuntu
```

Comment:
this may be a separate effort, but you can likely simplify the syntax for the PPAs to look [like this](https://github.com/canonical/charmed-mysql-snap/blob/8.0/edge/snap/snapcraft.yaml#L23-L35)